### ISSUE TITLE
Figure out weird behavior when sending lots of requests

### ISSUE THREAD

**Issue by benrudolph** [dimagi/loveseat#3]
Couch goes really slow when on a few of the docs when you specify lots of iterations

**Comment by dannyroberts** [comment]
You're sure it's not just couch being slow?

**Comment by benrudolph** [comment]
no, but [here](http://guide.couchdb.org/draft/performance.html) it says that couchdb should get faster the more reads you do

**Comment by benrudolph** [comment]
worth looking into at least. the numbers seem odd
